content/contents.lr

```diff
@@ -16,7 +16,7 @@ So if you're looking to integrate CC licenses or CC licensed works into your app
     <div class="card-img-top">
       <img class="mx-auto d-block" src="/comments-solid.png" alt="Community icon"/>
     </div>
-    <div class="card-body" align="center">
+    <div class="card-body">
```

Comment:
If you remove the `align` attribute, you have to add the corresponding CSS rule.

Reply:
Will do.

Reply:
✅ Done